src/popup/components/NameItem.vue

```diff
@@ -200,8 +200,10 @@ export default defineComponent({
     const error = ref(false);
     const pointerInput = ref();
 
+    const getDefaultName = store.getters['names/getDefault'] as (a?: string) => string | undefined;
```

Comment:
i think, this getter should be moved to utils function in this pr

Reply:
This makes no sense as next week we will replace Vuex "names" module with composable. So this line is temporary.